<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela</class>
 <widget class="QWidget" name="tela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="wid_main" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="wid_cima" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="wid_c_cima" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="add_rbtt">
              <property name="text">
               <string>Adicionar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="remover_rbtt">
              <property name="text">
               <string>Remover</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wid_c_baixo" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="cod_rbtt">
              <property name="text">
               <string>CÃ³digo Cliente</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="cnpj_rbtt">
              <property name="text">
               <string>CNPJ</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wid_meio" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="2">
          <widget class="QLineEdit" name="cliente_nome_ln"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="rca_nome_ln"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="cliente_lb">
           <property name="text">
            <string>Cod. Cliente</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="rca_cod_ln">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="rca_lb">
           <property name="text">
            <string>RCA</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="cliente_cod_ln">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="rca_tbtt">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="rcasub_lb">
           <property name="text">
            <string>RCA Substituir</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="cliente_tbtt">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="rcasub_cod_ln">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QToolButton" name="rcasub_tbtt">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="rcasub_nome_ln"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wid_baixo" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBorca3_ckboxx_3">
           <property name="text">
            <string>RCA3 da 302</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="rca2_ckbox">
           <property name="text">
            <string>RCA2 da 302</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="rca1_ckbox">
           <property name="text">
            <string>RCA1 da 302</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="usapar_ckbox">
           <property name="text">
            <string>Utiliza Par</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wid_botoes" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="esp_botoes">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="executar_btt">
           <property name="text">
            <string>Executar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fechar_btt">
           <property name="text">
            <string>Fechar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
